qa/tasks/cephfs/test_volumes.py

```diff
@@ -8190,3 +8190,100 @@ def test_volumes_module_finisher_thread(self):
 
         # verify trash dir is clean
         self._wait_for_trash_empty()
+
+class TestToggleVolumes(CephFSTestCase):
+    '''
+    Contains code for enabling/disabling mgr/volumes plugin.
+    '''
+
+    MGR_MOD_NAME = 'volumes'
+    CONFIRM = '--yes-i-really-mean-it'
+
+    def test_enabled_by_default(self):
```

Comment:
This doesn't test anything related to this change.

Reply:
Ideally, this test should've been already present but since it's not, I've added it to ensure that this PR doesn't affect this already existing functionality of volumes plugin.

Reply:
Marking this as resolved.